airflow/providers/pinecone/hooks/pinecone.py

```diff
@@ -24,7 +24,7 @@
 from functools import cached_property
 from typing import TYPE_CHECKING, Any
 
-from pinecone import Pinecone, PodSpec, ServerlessSpec
+from pinecone import Pinecone, PodSpec, ServerlessSpec, Vector
```

Comment:
If we are just using Vector for typing. We should move it under:
```
if TYPE_CHECKING:
    from pinecone import Vector
```

Reply:
changed it